### PR TITLE
Create Dota2lounge.com.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

**Comment by fuglede** [comment]
The site blocks necessary mixed content from http://cdn.dota2lounge.com/css/bright.min.css?1456821822 (which can not be secured).

**Comment by ekaris** [comment]
What should I do?

**Comment by ekaris** [comment]
@J0WI What should I do?

**Comment by J0WI** [comment]
You can rewrite `cdn.dota2lounge.com` to `dota2lounge.com` and test if this works in all cases.
Please also document this issue and also that `www.dota2lounge.com` does not exist (because usually it does).

**Comment by ekaris** [comment]
Good?

File: src/chrome/content/rules/Dota2lounge.com.xml [hunk 1]
```diff
@@ -1,15 +1,14 @@
 <!--
 	Non working:
 	www.dota2lounge.com
```
Comment (J0WI) [inline src/chrome/content/rules/Dota2lounge.com.xml]: Does not exist

File: src/chrome/content/rules/Dota2lounge.com.xml [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-
 -->
 <ruleset name="Dota2lounge.com">
 	<target host="dota2lounge.com" />
 
-	<rule from="^http:" to="https:" />
-
 	<test url="http://cdn.dota2lounge.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Dota2lounge.com.xml]: You  need to add this as a target host

**Comment by ekaris** [comment]
Good?

**Comment by fuglede** [comment]
This broke the site; the CSS will be blocked as mixed content even though we are securing it.

**Pull request merged**